esmvalcore/cmor/check.py

```diff
@@ -513,9 +514,9 @@ def _check_coord_values(self, coord_info, coord, var_name):
         if coord_info.valid_max:
             valid_max = float(coord_info.valid_max)
             if np.any(coord.points > valid_max):
-                if coord_info.standard_name == 'longitude' and \
-                        self.automatic_fixes:
-                    l_fix_coord_value = True
+                if coord_info.standard_name == 'longitude':
```

Comment:
I think moving the `and if self.automatic_fixes` from here to inside the function changes the logic in a way that is not correct: now it will never report a critical error message for longitude points that are out of range if automatic fixes are disabled.